kvm-install-vm

```diff
@@ -627,7 +627,7 @@ _EOF_
 
     outputn "Copying cloud image ($(basename ${IMAGE}))"
     DISK=${VMNAME}.qcow2
-    cp $IMAGE $DISK && ok
+    qemu-img create -f qcow2 -F qcow2 -b $IMAGE $DISK && ok
```

Comment:
What do you think about adding the quiet (`-q`) option here to keep the output formatting the same as before?

Reply:
Thanks for the input, I'll fix that.